schemas/NutritionSurvey.json

```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Nutrition Survey",
+  "description": "Data collection and quality assurance tools for Nutrition Surveys on mobile devices",
+  "type": "object",
+  "properties": {
+    "doc_type": {
+      "type": "string"
+    },
+    "uuid": {
+      "type": "string"
+    },
+    "syncDate": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "startTime": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "created": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "_id": {
+      "type": "string"
+    },
+    "_rev": {
+      "type": "string"
+    },
+    "modified": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "householdID": {
+      "type": "integer"
+    },
+    "cluser": {
+      "type": "integer"
+    },
+    "endTime": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "location": {
+      "type": "array",
+      "items": [
+        {
+          "type": "number"
+        },
+        {
+          "type": "number"
+        }
+      ],
+      "minItems": 2,
+      "maxItems": 2
+    },
+    "members": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/member"
```

Comment:
nope, i mean a external document in the database which stores reused data. for example if you are referencing to a team, have it it in a dedicate document in the database, in order to avoid duplicating information and enable navigation between linked documents